### PR TITLE
docs: Edge replication guide

### PR DESCRIPTION
**Pull request by johanandren**
On top of #1083 for now

### CODE REVIEW

File: akka-edge-docs/src/main/paradox/guide/5-charging-station.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+Java
+:  @@snip [DroneServiceImpl.java](/samples/grpc/local-drone-control-java/src/main/java/local/drones/DroneServiceImpl.java) { #charging }
+
+
+## Running
+
+The complete sample can be downloaded from GitHub, but note that it also includes the next steps of the guide:
+
+* Java: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-java
+* Scala: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-scala
```
Comment (patriknw) [inline akka-edge-docs/src/main/paradox/guide/5-charging-station.md:182]: These should be links to the zip files. I see that we still have a few github links remaining. Must be an oversight, can you change all of them to the zip files.

**Review by patriknw — COMMENTED** [review]
looked at the .md, looking good

File: akka-edge-docs/src/main/paradox/guide/5-charging-station.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+Scala
+:  @@snip [ChargingStation.scala](/samples/grpc/local-drone-control-scala/src/main/scala/charging/ChargingStation.scala) { #eventHandler }
+
+Java
+:  @@snip [ChargingStation.java](/samples/grpc/local-drone-control-java/src/main/java/charging/ChargingStation.java) { #eventHandler }
+
+### Serialization
+
+The state and events of the entity must be serializable because they are written to the datastore, if the local drone control needs to scale out across several nodes to handle traffic, the commands would also be sent between nodes within the Akka cluster. The sample project includes built-in CBOR serialization using the @extref[Akka Serialization Jackson module](akka:serialization-jackson.html). This section describes how serialization is implemented. You do not need to do anything specific to take advantage of CBOR, but this section explains how it is included.
```
[the remark this answers is not in the document]
Comment (johanandren) [reply akka-edge-docs/src/main/paradox/guide/5-charging-station.md:107]: This was copied from the first step of the guide, but I see we omit it for the entity in step 3 so I think we can drop it from here (and fix in step 1).

File: akka-edge-docs/src/main/paradox/guide/2-drone-location-to-cloud-service.md [hunk 3]
```diff
@@ -240,8 +240,12 @@ Java
...
 The complete sample can be downloaded from GitHub, but note that it also includes the next steps of the guide:
 
-* Java: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-java
-* Scala: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-scala
+* Scala
+    * [local-drone-control-service.zip](../attachments/local-drone-control-service-scala.zip)
+    * [restaurant-drone-deliveries-service.zip](../attachments/restaurant-drone-deliveries-service-scala.zip)
+* Java
+    * [local-drone-control-service.zip](../attachments/local-drone-control-service-java.zip)
+    * [restaurant-drone-deliveries-service.zip](../attachments/restaurant-drone-deliveries-service-java.zip)
```
Comment (patriknw) [inline akka-edge-docs/src/main/paradox/guide/2-drone-location-to-cloud-service.md:248]: no, both of them are included in same zip, which is named drone-scala.zip and drone-java.zip

File: akka-edge-docs/src/main/paradox/guide/3-restaurant-deliveries-service.md [hunk 4]
```diff
@@ -73,8 +73,8 @@ Java
 
 The complete sample can be downloaded from GitHub, but note that it also includes the next step of the guide:
 
-* Java: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-java
-* Scala: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-scala
+* Scala [restaurant-drone-deliveries-service.zip](../attachments/restaurant-drone-deliveries-service-scala.zip)
+* Java [restaurant-drone-deliveries-service.zip](../attachments/restaurant-drone-deliveries-service-java.zip)
```
Comment (patriknw) [inline akka-edge-docs/src/main/paradox/guide/3-restaurant-deliveries-service.md:77]: same here

File: akka-edge-docs/src/main/paradox/guide/4-local-drone-delivery-selection.md [hunk 5]
```diff
@@ -163,9 +163,13 @@ Java
...
 
-* Java: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-java
-* Scala: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-scala
-
+* Scala
+    * [local-drone-control-service.zip](../attachments/local-drone-control-service-scala.zip)
+    * [restaurant-drone-deliveries-service.zip](../attachments/restaurant-drone-deliveries-service-scala.zip)
+* Java
+    * [local-drone-control-service.zip](../attachments/local-drone-control-service-java.zip)
+    * [restaurant-drone-deliveries-service.zip](../attachments/restaurant-drone-deliveries-service-java.zip)
```
Comment (patriknw) [inline akka-edge-docs/src/main/paradox/guide/4-local-drone-delivery-selection.md:171]: and here

File: akka-edge-docs/src/main/paradox/guide/6-deploying-the-services.md [hunk 6]
```diff
@@ -116,8 +116,8 @@ We are now going to deploy the `restaurant-drone-deliveries-service` to the crea
 
 This step is for deploying:
 
-* Java: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-java
-* Scala: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-scala
+* Scala: [restaurant-drone-deliveries-service.zip](../attachments/restaurant-drone-deliveries-service-scala.zip)
+* Java: [restaurant-drone-deliveries-service.zip](../attachments/restaurant-drone-deliveries-service-java.zip)
```
Comment (patriknw) [inline akka-edge-docs/src/main/paradox/guide/6-deploying-the-services.md:120]: ^

File: akka-edge-docs/src/main/paradox/guide/6-deploying-the-services.md [hunk 7]
```diff
@@ -216,8 +216,8 @@ kubectl get services
 
 This step is for deploying:
 
-* Java: https://github.com/akka/akka-projection/tree/main/samples/grpc/local-drone-control-java
-* Scala: https://github.com/akka/akka-projection/tree/main/samples/grpc/local-drone-control-scala
+* Scala: [local-drone-control-service.zip](../attachments/local-drone-control-service-scala.zip)
+* Java: [local-drone-control-service.zip](../attachments/local-drone-control-service-java.zip)
```
Comment (patriknw) [inline akka-edge-docs/src/main/paradox/guide/6-deploying-the-services.md:220]: ^

File: akka-edge-docs/src/main/paradox/guide/5-charging-station.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (patriknw) [reply akka-edge-docs/src/main/paradox/guide/5-charging-station.md:97]: remove this now?

File: akka-edge-docs/src/main/paradox/guide/5-charging-station.md [hunk 8]
```diff
@@ -0,0 +1,269 @@
...
+
+Scala
+:  @@snip [ChargingStationServiceImpl.scala](/samples/grpc/restaurant-drone-deliveries-service-scala/src/main/scala/charging/ChargingStationServiceImpl.scala) { }
+
+Java
+:  @@snip [ChargingStationServiceImpl.java](/samples/grpc/restaurant-drone-deliveries-service-java/src/main/java/charging/ChargingStationServiceImpl.java) { }
+
+## Interacting with the charging station at the edge
+
+The local-drone-control service does not contain the full gRPC API for interaction but instead has a single `goCharge` method
```
Comment (patriknw) [inline akka-edge-docs/src/main/paradox/guide/5-charging-station.md:157]: there is also `CompleteCharge`

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**